StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/UpdatePaymentMethodViewModel.swift

```diff
@@ -50,11 +50,12 @@ class UpdatePaymentMethodViewModel {
         case .SEPADebit:
             return .Localized.manage_sepa_debit
         default:
-            fatalError("Updating payment method has not been implemented for \(paymentMethod.type)")
+            stpAssertionFailure("Updating payment method has not been implemented for \(paymentMethod.type)")
+            return nil
```

Comment:
To improve ergonomics when using these I think we should just use an empty string rather than nil. Also let's use assertionFailure instead of stpAssertion, I updated my comment shortly after I had originally suggested stpAssertion. We want these to fire in merchant apps.

/// A wrapper that only calls `assertionFailure` when the `ENABLE_STPASSERTIONFAILURE` compiler flag is set.
/// Use this for assertions that should not trigger in merchant apps.
@inlinable @_spi(STP) public func stpAssertionFailure